package.json

```diff
@@ -12,8 +12,9 @@
     "postinstall": "npm prune && npm run build",
     "prepush": "opt --out prepush --exec 'npm run validate'",
     "start": "webpack-dev-server --config webpack/starterApp.js --devtool eval --progress --colors --hot --history-api-fallback --open",
-    "test": "NODE_ENV=test karma start",
-    "test:watch": "npm test -- --auto-watch --no-single-run --reporters progress",
+    "test": "jest",
+    "test:watch": "npm test -- --verbose --notify --watch",
```

Comment:
@randycoulman I usually have tests running in a full height terminal window to right of Atom - this is really a matter of preference, but I think leaving the test command without arguments allows for the most easy usage outside of what is here.